Lib/selectors.py

```diff
@@ -357,7 +357,7 @@ def register(self, fileobj, events, data=None):
             poller_events |= self._EVENT_WRITE
         try:
             self._selector.register(key.fd, poller_events)
-        except Exception:
+        except:
```

Comment:
I'm guessing it matters if the original exception was `SystemExit`, `unregister` were to raise e.g. `KeyError`, and there was a `KeyError` exception handler active -- even though exceptions are chained, without the `finally` it would still raise an instance of `KeyError` which would be caught, rather than raising `SystemExit` which would not be caught. Then again such a `KeyError` handler would have to be considered overly broad (if it could catch exceptions from a `selector.register()` call).